kuma/static/styles/components/wiki/toc.scss

```diff
@@ -47,7 +47,6 @@ $padding-horizontal: 30px;
         color: $accent-light;
         display: inline-block;
         @include set-font-size(17px);
-        font-weight: bold;
```

Comment:
Verdana's bold is heavier than Open Sans. I pulled the bold here (and a couple other spots) to minimize the difference.